utilities/backupable/backupable_db.cc

```diff
@@ -647,6 +647,7 @@ Status BackupEngineImpl::Initialize() {
   // background
   for (int t = 0; t < options_.max_background_operations; t++) {
     threads_.emplace_back([this]() {
+      pthread_setname_np(pthread_self(), "backup_engine");
```

Comment:
I think it's glibc-specific, maybe conditionally compile this line using `#ifdef OS_LINUX`